src/bindings/jna/libelektra5j/src/main/java/org/libelektra/Key.java

```diff
@@ -476,7 +476,7 @@ public int getRef ()
 	/**
 	 * Tries to rewind the meta information for this key
 	 *
-	 * @retval 0 in case of no errors; 1 if key is not found; 2 if metakey is not found
+	 * @return 0 in case of no errors; 1 if key is not found; 2 if metakey is not found
```

Comment:
I assume, this change is because Javadoc doesn't support `@retval` the way Doxygen does?
If so, we should document somewhere that `@retval` shouldn't be used in the Java binding. (@lawli3t this may be relevant to your API reviews)

Reply:
Yes, i think we should only use javadoc compliant tags. I've added a note to src/bindings/jna/README.md